### DISCUSSION TITLE
Port blocking still required to stop being bricked ?

### DISCUSSION THREAD

**Discussion by gbjbasdw** [GerryDazoo/Slinger#651] [General]
Long time no contact.. Slinger still working brilliantly - once again thanks to Gerry.

I'm in the UK using a Virginmedia Superhub 2AC which does allow port blocking of port 5678 ( I think that was the one anyway ) to stop those nasty ( not the actual word I would have liked to use ) people at Slingmedia bricking my Slingbox.

At the moment I'm fine but I may have to move to a newer router from them due to the switch off of analogue landline phones which will mean I need a router with a phone socket on the back as I understand it. Sadly all the new Virginmedia have been dumbed down and don't have the port blocking facility anymore. Great !

I could run the Virginmedia router in modem mode and get my own router but that's two pieces of kit to maintain so IF don't really need to port block anymore then I don't have an issue.

So "Do I need to still block the Slingmedia port ?"

Cheers,

Steve

**Comment by GerryDazoo** [comment]
What model slingbox do you have? You may be able to get away with it if you have a 500/M1/M2 but it is still highly recommended. But the port blocking on the new router might be hidden under "parental controls". On my box you can block a particular MAC address from accessing the internet at certain times of day to keep the kiddies from surfing after bed time. Just block the slingbox MAC for 24/7.

**Reply by garabik** [reply]
Or, if you are using it only on your  LAN, use a bogus gateway. The side benefit of manual IP setup is that this will survive your router being reset to the defaults.

**Comment by garabik** [comment]
The server in question is 67.148.153.220 (although slingboxes do also DNS queries so that might change). I noticed that recently (e.g. within one month) the response when tested with telnet to 5678 changed - previously it waited, now the connection drops  almost immediately. But the server is still alive, and I am not going to test it with my slingbox...

**Reply by gbjbasdw** [reply]
I have now had to change my router to a Virgin Hub 3 which does not have port blocking.
The new router does however  have port forwarding for both local and external ports.
If I create a port forwarding rule that has the following would it stop me getting bricked ?

192.168.0.254 local ports 5678 to 5678 External ports 5678  to 5678 protocol TCP & UTP where the IP address is non-existent

I've tried to connect to my WAN address with ":5678" appended to it and it comes back as unreachable.

I have a Pro HD box and I will need to use it when I am away from home. BTW

Any thoughts ?

**Reply by GerryDazoo** [reply]
As far as I know it's still needed but I'm not willing to brick my working box to verify.
You need to block OUTGOING connections from your slingbox to the internet. Port forwarding won't do the trick.
Most modern routers have outgoing connection blocking capabilities but they are sometimes called something else. Look for "Parental COntrols" or something similar. A feature used to block internet access for kiddos after bedtime. Just block the slingbox for 24/7.
